src/client.ts

```diff
@@ -287,10 +290,13 @@ export class WT {
     fetch(`${this.getRoot()}/wt/t`, {
       method: 'POST',
       credentials: 'include',
+      body: JSON.stringify(payload),
       headers: {
-        'Content-Type': 'application/x-www-form-urlencoded',
+        'Content-Type': 'text/plain',
+        ...this.wtConfig.fetchConfig?.headers,
       },
-      body: query,
+      keepalive: true,
```

Comment:
Aha - your callout from earlier. To clarify, this'll mean more `pageview` events possibly?

Reply:
this is a step toward capturing events that occur when leaving a page - there's an additional enhancement required to flush events when navigating away from a page. i need to do a tiny bit more research on the correct way to do that as there are some new-ish updates regarding browser back/forward caching that i want to understand before adding to this library directly